extensions/ql-vscode/src/databases/code-search-api.ts

```diff
@@ -26,13 +27,16 @@ export async function getCodeSearchRepositories(
       per_page: 100,
     },
   )) {
+    i++;
     nwos.push(...response.data.map((item) => item.repository.full_name));
-    // calculate progress bar: 80% of the progress bar is used for the code search
-    const totalNumberOfRequests = Math.ceil(response.data.total_count / 100);
-    // Since we have a maximum of 1000 responses of the api, we can use a fixed increment whenever the totalNumberOfRequests would be greater than 10
-    const increment =
-      totalNumberOfRequests < 10 ? 80 / totalNumberOfRequests : 8;
-    progress.report({ increment });
+    const totalNumberOfResults = Math.ceil(response.data.total_count / 100);
```

Comment:
Isn't this still the total number of requests rather than results? Maybe something like `totalNumberOfResultPages`?